sdk/loadtestservice/Azure.Analytics.LoadTestService/src/GlobalSuppressions.cs

```
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

I don't think we should suppress these error. They lack of justification.